forms-flow-bpm/forms-flow-bpm-camunda/src/main/resources/META-INF/resources/services/org.camunda.connect.spi.ConnectorProvider

```diff
@@ -0,0 +1 @@
+org.camunda.bpm.extension.hooks.adapter.mail.CustomConnectorProviderPlugin
```

Comment:
can you please change the file path & check?
It should be  META-INF/services/org.camunda.connect.spi.ConnectorProvider instead of  META-INF/resources/services/org.camunda.connect.spi.ConnectorProvider.

Reply:
sure. Thanks